kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/modem/sierra/mc87xx/SierraMc87xx.java

```diff
@@ -666,4 +667,34 @@ public void disableDiversity() throws KuraException {
     public String getFirmwareVersion() throws KuraException {
         return "N/A";
     }
+
+    @Override
```

Comment:
We should avoid to return null (at least an "N/A") in method without implementation, here and below.

Reply:
OK